src/Database/EloquentBuilder.php

```diff
@@ -93,11 +93,18 @@ public function whereIn($column, $values, $boolean = 'and', $not = false)
             return $this;
         }
 
+        if ($where['type'] === 'InSub') {
```

Comment:
```suggestion
        if ($where['type'] === 'InSub' || $where['type'] === 'NotInSub') {
```

Don't we need to handle that, too?